appveyor.yml

```diff
@@ -6,7 +6,7 @@ branches:
     - /release.*/
     - appveyor
     - hunter
-clone_depth: 1
+clone_depth: 100
```

Comment:
Is this really needed?

Reply:
To allow git access previous version tags `v0.X.Y...`.

Reply:
And what if a branch has like 200 commits?

This isn't foolproof. Only by cloning the whole repo you can be sure...

Reply:
That's true, but it will still build. Just the project version will not have the distance from the latest release.